product_docs/docs/pgd/5/reference/commit-scopes.mdx

```diff
@@ -157,11 +161,11 @@ The value `group` means the preceding `commit_scope_group` specification also af
 
 The value `partner` means the partner node decides whether transactions can be committed. This value is allowed only on groups with 2 data nodes. 
 
-The value `raft` means the COMMIT decision is done using Raft consensus of all the nodes in the cluster, independent of any `commit_scope_group` consensus. This option has low performance and is for backward compatibility only.
-
-See ["Commit decisions" in Group Commit](../durability/group-commit/#commit-decisions).
+The value `raft` means the COMMIT decision is noted using Raft consensus of all nodes, after the nodes in the `commit_scope_group` have confirmed. This ensures that all nodes have a consensus on the decision. 
```

Comment:
This wording is really weird

Reply:
The value `raft` uses PGD's built-in Raft consensus. Once all the nodes in the selected commit scope group have confirmed the transaction, to ensure that all the nodes in the PGD cluster have noted the transaction, it is noted with the all-node Raft.

Any less weird?

Reply:
(Just committed that)

Reply:
Yes this is much better, thanks